tests/corrupted_stamp_test.cpp

```diff
@@ -76,82 +77,89 @@ std::vector<headers_test_case> marginal_cases = {
 	// data_size <= disk_size
 	{0, 1, 100, 99, DNET_EXT_VERSION_V1, 0, {0}, {0}},
 	{0, 1 << 9, 100, 42, DNET_EXT_VERSION_V1, 0, {0}, {0}},
-
 	// disk_size == 0
 	{0, 1, 0, 99, DNET_EXT_VERSION_V1, 0, {0}, {0}},
-	// version != 1
-	{0, 1, 1, 99, 0, 0, {0}, {0}},
-	{0, 1, 1, 99, 2, 0, {0}, {0}},
+
+	// version isn't equal to DNET_EXT_VERSION_V1 or DNET_EXT_VERSION_FIRST
+	{0, BLOB_DISK_CTL_EXTHDR, 1, 99, 2, 0, {0}, {0}},
 	// timestamp out of range.
-	{0, 1, 100, 101, DNET_EXT_VERSION_V1, DNET_SERVER_SEND_BUGFIX_TIMESTAMP + 1, {0}, {0}},
-	{0, 1, 100, 101, DNET_EXT_VERSION_V1, DNET_SERVER_SEND_BUGFIX_TIMESTAMP + 42, {0}, {0}},
-	{0, 1, 100, 101, DNET_EXT_VERSION_V1, DNET_SERVER_SEND_BUGFIX_TIMESTAMP + 100500, {0}, {0}},
```

Comment:
Why were they removed?

Reply:
No difference between `DNET_SERVER_SEND_BUGFIX_TIMESTAMP + 1` and `DNET_SERVER_SEND_BUGFIX_TIMESTAMP + 42`. It's the only case enough: `time` is greater than `DNET_SERVER_SEND_BUGFIX_TIMESTAMP`. No benefit from repeating of the equal cases. Just the run-time grows. Tests must run fast.

Reply:
But I can revert.